tests/Application/Kernel.php

```diff
@@ -5,24 +5,14 @@
 namespace Tests\Setono\SyliusFeedPlugin\Application;
```

Comment:
You could copy the file from here: https://github.com/Setono/SyliusPluginSkeleton/blob/master/tests/Application/Kernel.php